build.sh

```diff
@@ -79,6 +84,19 @@ then
   $cmd || tcStatus 1 "Unit-Testing failed"
 fi
 
+tcLog "Checking for skip-ci labels"
+computeFastBuild
+if [ -n "$FAST_BUILD" ]
```

Comment:
I think it would be more idiomatic if `computeFastBuild` returned `0` for a fast build or `1` otherwise. Then the test would become
```sh
if computeFastBuild
then
...
```
To achieve that, the easiest way in my opinion would be to replace the last line of the function with
```sh
echo "$prMessages" | grep -q  '\[skip ci\]' && return 0 || return 1
```